Ivan Alenin - request fo 1-st 2-th lesson

aiscorp opened the pull request.

aiscorp committed:
Initial commit with template Rect code and 1-5 tasks

aiscorp committed:
Solved a lot of problems, added components, many troubles with binding between components.

aiscorp commented:
1. Прошу посмотреть в чем у меня ошибка сейчас с классовыми компонентами: они должны передавать между собой текстовое сообщение через родителя, но у меня никак не получается понять в чем причина падений. 
2. В текущей реализации InputBlock получает метод чтения сообщения от родителя App, затем App формирует состояние MessageGroup в зависимости тот того есть сообщение или нет, устанавливая в true флаг наличия сообщения. Не работает, частями работает как нужно. Где-то тупые ошибки.

aiscorp committed:
At most it's working now..
https://yadi.sk/i/VHFv1BSEyCL7Wg

aiscorp commented:
Удалось добиться задуманного содержания.
Будет интересно получить ваше ревью и критику.

aiscorp committed:
Folder '.idea' deleted from pull request

aiscorp committed:
UI improvements and short code refactoring.
Deployment with sftp on https://ais-code.ru/bot/

SerSerch commented on IvanAlenin/package.json:
```diff
@@ -0,0 +1,44 @@
+    "autoprefixer": "^9.8.0",
+    "bootstrap": "^4.5.0",
+    "grep": "^0.1.0",
+    "html-inline-css-webpack-plugin": "^1.8.0",
```
autoprefixer
grep
html-inline-css-webpack-plugin
в devDependencies

idoleg reviewed (changes requested):
Необходимо отменить изменения .gitignore

aiscorp committed:
Merge branch 'master' of https://github.com/idoleg/react-01.06

 Conflicts:
	.gitignore

aiscorp committed:
UI improvements and short code refactoring.
Deployment with sftp on https://ais-code.ru/bot/

aiscorp commented:
> Необходимо отменить изменения .gitignore

Исправлено это и правки выше.

aiscorp committed:
Folders for components, chat's block scrolling to the bottom.

aiscorp committed:
1.Part ~50 of the chat's UI is done: new tabs, scrolling, flexibility for pc and mobile.
https://yadi.sk/i/perWjnMKF2ORIQ
2. Prod: sss external export, image loader

aiscorp committed:
1. Main page Ui 100% (html+css).
2. It's a fact that refactoring of bad react components code is needed. This's next stage.
